bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -113,14 +113,7 @@ spec:
             set +o allexport
           fi
 
-          # Determine the ovn rundir.
-          if [[ -f /usr/bin/ovn-appctl ]] ; then
-              # ovn-appctl is present. Use new ovn run dir path.
-              OVNCTL_PATH=/usr/share/ovn/scripts/ovn-ctl
-          else
-              # ovn-appctl is not present. Use openvswitch run dir path.
-              OVNCTL_PATH=/usr/share/openvswitch/scripts/ovn-ctl
-          fi
+          OVNCTL_PATH=/usr/share/ovn/scripts/ovn-ctl
```

Comment:
Can you just get rid of the variables and substitute the paths in directly? Then we don't have to hunt around multiple places in the file to figure out where things are actually ending up.

Reply:
Yeah, +1

Reply:
OK